TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function-test-01.xml

```diff
@@ -100,14 +100,22 @@
     </testCase>
 
     <testCase id="decision_011">
-        <description>null value stored in context</description>
-        <resultNode name="decision_011" type="decision">
+        <description>null map and key params</description>
+        <resultNode errorResult="true" name="decision_011" type="decision">
             <expected>
                 <value xsi:nil="true"></value>
             </expected>
         </resultNode>
     </testCase>
 
+    <testCase id="decision_012">
+        <description>null value stored in context</description>
```

Comment:
the original submission for 

> null value stored in context

shall refer, I believe, to test case:

https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function.dmn#L123

which is decision logic decision `decision_012 `.

Created accordingly and aligned, a non-error test case where the expected result is the `null` stored in the context entry for `"a"`.